File: entregavel1/js/a_numeroPrimo.js
function numPrimo() {
    let num = prompt("Digite um número e veja se ele é primo...");
    let contDivs = 0;

    for (let index = 1; index <= num; index++) {
        if(num%index === 0) {
            contDivs = contDivs + 1;
        }
    }

    contDivs > 2 ? alert("Não é primo.") : alert("É primo...");

}

File: entregavel1_java/classes/A_NumeroPrimo.java
package entregavel1_java.classes;

import java.util.Scanner;

public class A_NumeroPrimo {

    public static String numPrimo() {
        Scanner sc1 = new Scanner(System.in);
        String ehPrimo = "";

        System.out.println("Digite um número e veja se ele é primo...");
        int num = sc1.nextInt();
        int contDivs = 0;
        sc1.close();

        for (int index = 1; index <= num; index++) {
            if(num%index == 0){
                contDivs = contDivs + 1;
            }
        }

        if(contDivs > 2) {
            ehPrimo = "Não é primo.";
        } else {
            ehPrimo = "É primo.";
        }

        return ehPrimo;
    }
    
}